packages/@wdio_electron-utils/test/index.spec.ts

```diff
@@ -491,33 +524,82 @@ describe('getAppBuildInfo', () => {
     });
   });
 
-  it('should return the expected config for a builder dependency with inline config', async () => {
-    const packageJsonPath = getFixturePackagePath('builder-dependency-inline-config');
+  it('should try to access all variants of builder config files', async () => {
+    const packageJsonPath = getFixturePackagePath('builder-dependency-no-config');
     const packageJson = JSON.parse(await fs.readFile(packageJsonPath, 'utf-8'));
-    await expect(
+    const spy = vi.spyOn(fs, 'access');
+    try {
       await getAppBuildInfo({
         packageJson,
         path: packageJsonPath,
-      }),
-    ).toStrictEqual({
-      appName: 'builder-dependency-inline-config',
-      config: { productName: 'builder-dependency-inline-config' },
-      isBuilder: true,
-      isForge: false,
+      });
+    } catch (_e) {
+      // An error will always occur because the configuration file does not exist.
+      // Ignore errors to test to ensure all expected filename patterns are covered
+    }
+    const accessedFilenames = spy.mock.calls.map((call) => {
+      if (typeof call[0] === 'string') {
+        return path.basename(call[0]);
+      }
+      assert.fail('Unexpected call with parameter type of Buffer');
```

Comment:
Just a small nit, how do we know the type is Buffer?  Would the below message be more appropriate?

```suggestion
      assert.fail(`fs.access called with a non-string value: ${call[0]}`);
```